src/js/utils/helpers.js

```diff
@@ -603,6 +617,22 @@ define([
         element.className = strings.trim(_.difference(originalClasses, removeClasses).join(' '));
     };
 
+    utils.toggleClass = function (element, c, toggleTo) {
+        if(_exists(toggleTo)) {
```

Comment:
I think it'd be worth adding a short-circuit for performance
    
    var hasClass = utils.hasClass(element, c);
    toggleTo = _.isBoolean(toggleTo) ? toggleTo : !hasClass;
    
    // short circuit if nothing to do
    if (toggleTo === hasClass) {
        return;
    }
    
    // now toggle